### DISCUSSION TITLE
AdGuard cannot be updated easily like before !

### DISCUSSION THREAD

**Discussion by kingtr0n** [AdguardTeam/AdGuardHome#4352] [Q&A]
I have a problem in updating AdGuard, it does not update when there is a new version. I remember before we were be able to click "update" and get it updated, now you can only see the message "AdGuard Home v0.107.4 is now available! Click here for more info." and that is it. any idea ?
btw I'm using v0.107.3 AdGuardHome_linux_mipsle_softfloat
installed on ubiquiti edgerouter er-x using  this guide: https://dev.to/zer0ed/install-adguard-home-on-edgerouter-x-including-local-hostname-resoluion-using-dnsmasq-2hmc

<img width="446" alt="error" src="https://user-images.githubusercontent.com/100774251/156363888-72a200e6-be30-40d6-b000-fb0bbbba3bf9.png">

**Comment by hl2guide** [comment]
same for me:

![example](https://user-images.githubusercontent.com/17044982/156865600-2abe6984-b936-43ae-afdc-89598e6e7f1b.png)

Auto update has not worked for 3 versions for me.

**Reply by kingtr0n** [reply]
@hl2guide thanks

in your case the update has started and then failed, in my case there is no option to start the update !! "only a notification"
